sparkmagic/sparkmagic/tests/test_reliablehttpclient.py

```diff
@@ -232,3 +232,4 @@ def test_kerberos_auth_check_auth():
     client = ReliableHttpClient(endpoint, {}, retry_policy)
     assert_is_not_none(client._auth)
     assert isinstance(client._auth, HTTPKerberosAuth)
+
```

Comment:
Adding a test here to see that the verify parameter in the request is honoring the value of `conf.custom_certfiles_path()` when it's set would be useful and would have caught the lack of method call I mentioned in the other file. Let's add it.